src/theme/colors.ts

```diff
@@ -28,6 +28,7 @@ const colors = {
   'navyblue-800': '#0F1623',
   'navyblue-900': '#0A0F1A',
 
+  'blue-400-10': '#E6EFFA', // 10% of blue-400
```

Comment:
`[discussion]` I think instead of adding new colors in this fashion we should be using either the opacity variants that @3nvi added or perhaps a combination of the `blue-400` + `10% opacity` + `white` background.

Either that or there's a `lightenDarkenColor` utility here in pounce you may be able to use on the alert bgs to get the proper mix of color + white